core/src/integrationTest/kotlin/ch/ergon/dope/integrationTest/TestCouchbaseDatabase.kt

```diff
@@ -0,0 +1,94 @@
+package ch.ergon.dope.integrationTest
+
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.fromable.UnaliasedBucket
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.NumberType
+import ch.ergon.dope.validtype.StringType
+import com.couchbase.client.kotlin.Cluster
+import com.couchbase.client.kotlin.query.execute
+import kotlinx.coroutines.runBlocking
+import org.testcontainers.couchbase.BucketDefinition
+import org.testcontainers.couchbase.CouchbaseContainer
+import org.testcontainers.utility.DockerImageName
+import kotlin.test.assertEquals
+import kotlin.time.Duration.Companion.seconds
+
+const val BUCKET = "testBucket"
+
+object TestCouchbaseDatabase {
+    private val container = CouchbaseContainer(
+        DockerImageName.parse("couchbase/server:community-7.6.2"),
+    )
+    val cluster: Cluster
+    val testBucket = UnaliasedBucket(BUCKET)
+    val idField = Field<NumberType>("id", testBucket.name)
+    val typeField = Field<StringType>("type", testBucket.name)
+    val nameField = Field<StringType>("name", testBucket.name)
+    val isActiveField = Field<BooleanType>("isActive", testBucket.name)
+    val orderNumberField = Field<StringType>("orderNumber", testBucket.name)
+    val deliveryDateField = Field<StringType>("deliveryDate", testBucket.name)
+
+    init {
+        initContainer()
+        cluster = Cluster.connect(
+            container.connectionString,
+            container.username,
+            container.password,
+        )
+        initDatabase()
+    }
+
+    fun resetDatabase() {
+        runBlocking {
+            cluster.waitUntilReady(15.seconds).query("DELETE FROM $BUCKET").execute()
+        }
+        initDatabase()
+    }
+
+    private fun initContainer() {
+        container.withBucket(BucketDefinition(BUCKET))
+        container.start()
+    }
+
+    private fun initDatabase() {
+        val collection = cluster.bucket(BUCKET).defaultCollection()
+        tryUntil {
+            runBlocking {
+                cluster.waitUntilReady(15.seconds)
+                for (i in 1..5) {
```

Comment:
consider using: 
(0..3).forEach { index -> }
instead of the for loop

Reply:
done